### PR TITLE
Implement a -f option for commands which may need to tolerate

### PR DESCRIPTION
**Pull request by davidewatson**
depreciated configuration files (#66).

### CODE REVIEW

File: cmd/cluster.go [hunk 1]
```diff
@@ -47,4 +48,10 @@ func init() {
 		"p",
 		"",
 		"registry password")
+	clusterCmd.PersistentFlags().BoolVarP(
+		&configForced,
+		"force",
+		"f",
+		false,
+		"true if operation should be proceed even if config is depreciated (default false)")
```
Comment (DStorck) [inline cmd/cluster.go]: I'm going to pull a Joe here and get nit-picky about spelling (since he left for the day). Should be deprecated, not depreciated.

**Comment by coffeepac** [comment]
is there a unit test you can write for this?  might need to refactor how we are building the commands we pass into the containers so we can check what the command is before executing it.

if its a huge PITA its not required.

**Comment by davidewatson** [comment]
Will do.

**Comment by davidewatson** [comment]
I think doing this in a meaningful way will require refactoring the argument parsing for all commands. Any objections to handling this along with https://github.com/samsung-cnct/k2cli/issues/46 ?

**Comment by coffeepac** [comment]
@davidewatson sounds fine to me.

**Pull request merged**